support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButtonContainer.jsx

```diff
@@ -30,49 +31,35 @@ type PropTypes = {|
   otherAmounts: OtherAmounts,
 |};
 
-
-const setupStripe = (setStripeHasLoaded: () => void) => {
-  const htmlElement = document.getElementById('stripe-js');
-  if (htmlElement !== null) {
-    htmlElement.addEventListener(
-      'load',
-      setStripeHasLoaded,
-    );
-  }
-};
-
 // ----- Component ----- //
 
-function StripePaymentRequestButtonContainer(props: PropTypes) {
+class StripePaymentRequestButtonContainer extends React.Component<PropTypes, void> {
```

Comment:
This needs to be a class because we need to call `setStripeHasLoaded` on `componentDidMount`